src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age ? (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    ) : ''
+    }
+
+    {person.isMarried === true ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my `}
+        {person.sex === 'm'
+          ? 'wife'
+          : 'husband'
+        }
```

Comment:
Create a variable for wife/husband
![image](https://github.com/mate-academy/react_person/assets/96409563/96767611-d0c8-4301-9192-770fee7a5255)